.github/workflows/android.yml

```diff
@@ -0,0 +1,70 @@
+name: android
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    if: github.event.pull_request.draft == false
+
+    strategy:
+      matrix:
+        arch: [armeabi-v7a, arm64-v8a, x86, x86_64]
+
+    runs-on: ubuntu-latest
+    container: ghcr.io/f3d-app/f3d-ci
+
+    env:
+      NDK_VERSION: r21e
+      API_LEVEL: 27
```

Comment:
for my own culture, what is android API level ?

Reply:
It's the target Android version. Tbh the value is a bit random right now and we'll probably have to adjust it.  
27 is Android 8.1 and covers 88.4% of the devices.  
https://apilevels.com